The terraform-provider-grafana_v1.40.1 plugin crashed!

### Terraform Version
* Terraform: v1.4.6 on darwin_arm64
* Terraform Grafana Provider: v1.40.1
* Grafana: 8.1.8

### Affected Resource(s)
* grafana_dashboard
* terraform_maintained

### Terraform Configuration Files
Default configuration from https://registry.terraform.io/providers/hashicorp/google/latest/docs

### Debug/Panic Output
```
Stack trace from the terraform-provider-grafana_v1.40.1 plugin:

panic: interface conversion: interface {} is nil, not []interface {}

goroutine 16 [running]:
github.com/grafana/terraform-provider-grafana/internal/resources/grafana.NormalizeDashboardConfigJSON({0x10377f520?, 0x140001bd770?})
        github.com/grafana/terraform-provider-grafana/internal/resources/grafana/resource_dashboard.go:366 +0x370
github.com/grafana/terraform-provider-grafana/internal/resources/grafana.ReadDashboard({0x1035779b0?, 0x140004d9230?}, 0x14000651700, {0x103717d20?, 0x140006476d0?})
        github.com/grafana/terraform-provider-grafana/internal/resources/grafana/resource_dashboard.go:257 +0x81c
github.com/grafana/terraform-provider-grafana/internal/provider.addResourcesMetadataValidation.func1({0x10388b508, 0x14000269860}, 0x0?, {0x103717d20, 0x140006476d0})
        github.com/grafana/terraform-provider-grafana/internal/provider/provider_validation.go:60 +0xe0
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x140000f1ea0, {0x10388b540, 0x14000269110}, 0xd?, {0x103717d20, 0x140006476d0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema/resource.go:724 +0xe8
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0x140000f1ea0, {0x10388b540, 0x14000269110}, 0x14000620750, {0x103717d20, 0x140006476d0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema/resource.go:1015 +0x46c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0x140003f3e90, {0x10388b540?, 0x14000268ff0?}, 0x140004ce700)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.26.1/helper/schema/grpc_provider.go:613 +0x400
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0x140002195e0, {0x10388b540?, 0x14000675d70?}, 0x14000418600)
        github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/tf5server/server.go:748 +0x3e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x10384b8a0?, 0x140002195e0}, {0x10388b540, 0x14000675d70}, 0x1400064c850, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:349 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0x140002141e0, {0x10388f4d8, 0x140003731e0}, 0x14000154480, 0x14000400ae0, 0x103e12d30, 0x0)
        google.golang.org/grpc@v1.54.0/server.go:1345 +0xc90
google.golang.org/grpc.(*Server).handleStream(0x140002141e0, {0x10388f4d8, 0x140003731e0}, 0x14000154480, 0x0)
        google.golang.org/grpc@v1.54.0/server.go:1722 +0x82c
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.54.0/server.go:966 +0x84
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.54.0/server.go:964 +0x28c

Error: The terraform-provider-grafana_v1.40.1 plugin crashed!
```




### Expected Behavior
What should have happened?

Terraform import should have imported the resource

### Actual Behavior
What actually happened?
```
╷
│ Error: Request cancelled
│ 
│ The plugin.(*GRPCProvider).ReadResource request was cancelled.
╵
```


### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create Grafana dashboard using Terraform
2. run `terraform import grafana_dashboard.terraform_maintained GC797v0Vz`

If you need any more information, please let me know.
Thanks!
Adrien

--- a/internal/resources/grafana/resource_dashboard.go
+++ b/internal/resources/grafana/resource_dashboard.go
@@ -362,7 +362,7 @@ func NormalizeDashboardConfigJSON(config interface{}) string {
 	// from the dashboard JSON other than "name" or "uid".
 	// Grafana will populate all other libraryPanel attributes, so delete them to avoid diff.
 	panels, hasPanels := dashboardJSON["panels"]
-	if hasPanels {
+	if hasPanels && panels != nil {
 		for _, panel := range panels.([]interface{}) {
 			panelMap := panel.(map[string]interface{})
 			delete(panelMap, "id")

--- a/internal/resources/grafana/resource_dashboard.go
+++ b/internal/resources/grafana/resource_dashboard.go
@@ -361,9 +361,8 @@ func NormalizeDashboardConfigJSON(config interface{}) string {
 	// similarly to uid removal above, remove any attributes panels[].libraryPanel.*
 	// from the dashboard JSON other than "name" or "uid".
 	// Grafana will populate all other libraryPanel attributes, so delete them to avoid diff.
-	panels, hasPanels := dashboardJSON["panels"]
-	if hasPanels && panels != nil {
-		for _, panel := range panels.([]interface{}) {
+	if panels, ok := dashboardJSON["panels"].([]interface{}); ok {
+		for _, panel := range panels {
 			panelMap := panel.(map[string]interface{})
 			delete(panelMap, "id")
 			if libraryPanel, ok := panelMap["libraryPanel"].(map[string]interface{}); ok {

